messages/commands/transfer.go

```diff
@@ -28,3 +32,34 @@ type DeclineTransfer struct {
 	Amount        int64
 	Reason        messages.DebitFailureReason
 }
+
+// MessageDescription returns a human-readable description of the message.
+func (m *Transfer) MessageDescription() string {
+	return fmt.Sprintf(
+		"transfering %s from account %s to account %s",
+		messages.FormatAmount(m.Amount),
+		m.FromAccountID,
+		m.ToAccountID,
+	)
+}
+
+// MessageDescription returns a human-readable description of the message.
+func (m *ApproveTransfer) MessageDescription() string {
+	return fmt.Sprintf(
+		"approving transfer of %s from account %s to account %s",
+		messages.FormatAmount(m.Amount),
+		m.FromAccountID,
+		m.ToAccountID,
+	)
+}
+
+// MessageDescription returns a human-readable description of the message.
+func (m *DeclineTransfer) MessageDescription() string {
+	return fmt.Sprintf(
+		"declining transfer of %s from account %s to account %s for reason %s",
```

Comment:
For these ones I think it would be better to add a `DebitFailureReason.String()` function that returns a human-readable description of the reason that looks right in a sentence.   This could then be reused in the `DeclineWithdrawal` command as well.

The overall format of the message could use the Go error "convention" of tacking descriptions on the end after a colon, like:

```suggestion
		"declining transfer of %s from account %s to account %s: <reason goes here>,
```

Reply:
I've attempted this suggestion (see changes), but i'm not sure how to better word the human readable messages...

Reply:
The wording you've used there is fine, but I do see now that it's not really much different to what the enum value itself is. Maybe we should just remove the hyphens from the enum values.

I am pretty sure we chose to use strings (rather than using an `int` type with `iota) so that if you're playing around with the project you can see the actual meaning easily.

Reply:
Is it normal/ok to use plain text strings for enums like that?

Reply:
It's ok - it's probably not normal, but I think for an example it's very simple. The more code we add that isn't showing how to "do dogma" the harder it is to digest.

Reply:
I have made this change.